### PR TITLE
feat: Added feature for user to flip board #5

### PR DESCRIPTION
**Pull request by Prashanth-Chandra**
I have added a separate file which contains the function for flipping the board and added a button similar to the "Paste FEN" button for invoking the function

### CODE REVIEW

**Comment by SrNightmare09** [comment]
@Prashanth-Chandra have you made sure the changes you have made do not break the website and work as intended?

**Comment by Prashanth-Chandra** [comment]
Yes. I made sure that I did not touch any previous features and it works as intended.
The button will show the board from the opponents perspective (i.e. flipped by 180°)

**Comment by Prashanth-Chandra** [comment]
We can tweak also easily tweak the code (just changing the looping variables) to get the current boards FEN instead of reversing it.

File: src/flipBoard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            }
+            if (nullCount != 0 && j == 0) {
+                fen += nullCount;
+            }
+        }
+        if (i != 0) {
+            fen += '/';
+        }
+    }
+    console.log(fen);
```
Comment (SrNightmare09) [inline src/flipBoard.js:44]: Please remove any log messages you might have included for testing purposes.

**Review by SrNightmare09 — CHANGES_REQUESTED** [review]

**Comment by Prashanth-Chandra** [comment]
Removed the log messages.

**Comment by Prashanth-Chandra** [comment]
Done.

**Comment by SrNightmare09** [comment]
Thanks for the contribution! @Prashanth-Chandra

**Pull request merged**